Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR /bikes_server
+COPY . /bikes_server/
+RUN python3 -m venv venv \
+    && venv/bin/pip3 install -r requirements.txt \
+    && export BIKES_SQLITE=1 \
```

Comment:
Isn't this line unnecessary when you specify this exact variable in the `compose.yml` file?

Reply:
It is necessary, because the variable specified in `docker-compose.yml` is only for running the container and it has to be set here as well for running migrations